sample/src/utils.js

```diff
@@ -0,0 +1,13 @@
+var RSUtils = {
+  'getDateString': function getDateString() {
+    var date = new Date();
+    var dateString =
+        date.getFullYear() +
+        ('0' + (date.getMonth() + 1)).slice(-2) +
+        ('0' + date.getDate()).slice(-2) +
+        ('0' + date.getHours()).slice(-2) +
+        ('0' + date.getMinutes()).slice(-2) +
+        ('0' + date.getSeconds()).slice(-2);
+    return dateString;
+  }
+};

```

Comment:
This file's line ending seems odd to me(although it cannot be seen here), it seems we'd submit the patch with Unix-like line endding format.